ddmrp/views/product_adu_calculation_method_view.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0"?>
+<openerp>
+    <data>
+
+        <record model="ir.ui.view"
+                id="view_product_adu_calculation_method_tree">
+            <field name="name">product.adu.calculation.method.tree</field>
+            <field name="model">product.adu.calculation.method</field>
+            <field name="arch" type="xml">
+                <tree string="ADU Calculation Method">
+                    <field name="name"/>
+                </tree>
+          </field>
+        </record>
+        <record model="ir.ui.view"
+                id="view_product_adu_calculation_method_form">
+            <field name="name">product.adu.calculation.method.form</field>
+            <field name="model">product.adu.calculation.method</field>
+            <field name="arch" type="xml">
+                <form string="ADU Calculation Method">
+                    <group name="name">
+                        <field name="name"/>
+                    </group>
+                    <group name="method">
+                        <field name="method"/>
+                        <field name="use_estimates"/>
+                        <label for="horizon"/>
+                        <div name="horizon">
+                            <field name="horizon" class="oe_inline"/> days
```

Comment:
As remarked before, I suggest to make this field required when _horizon_ is _past_ or _future_:
`
<field name="horizon" class="oe_inline" attrs="{'required':[('method','in',['past','future'])]}"/> days`